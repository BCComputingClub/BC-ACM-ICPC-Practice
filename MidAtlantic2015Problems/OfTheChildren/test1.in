4
4
12
0 1 5
0 2 10
2 3 8
2 1 6
-1 -1 -1
4
4
12
0 1 5
0 2 10
2 3 8
2 1 6
0 3 6
-1 -1 -1
8
500
500
1000
1000
1000
1000
0 6 1009
0 4 1013
6 5 1019
4 3 1021
5 3 5000
5 2 997
3 1 991
2 7 983
1 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 500
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 550
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1000
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1000
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1025
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1051
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 950
6 7 978
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 2
1 3 3
1 4 4
2 3 5
2 4 6
3 4 1
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 3
1 3 4
1 4 5
2 3 6
2 4 1
3 4 2
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 4
1 3 5
1 4 6
2 3 1
2 4 2
3 4 3
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 5
1 3 6
1 4 1
2 3 2
2 4 3
3 4 4
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 6
1 2 6
1 3 1
1 4 2
2 3 3
2 4 4
3 4 5
-1 -1 -1
6
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
-1 -1 -1
11
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
-1 -1 -1
16
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
10 11 1
14 15 100
11 12 3
11 13 4
11 14 5
12 13 6
12 14 1
13 14 2
-1 -1 -1
21
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
10 11 1
14 15 100
11 12 3
11 13 4
11 14 5
12 13 6
12 14 1
13 14 2
15 16 1
19 20 100
16 17 4
16 18 5
16 19 6
17 18 1
17 19 2
18 19 3
-1 -1 -1
0
