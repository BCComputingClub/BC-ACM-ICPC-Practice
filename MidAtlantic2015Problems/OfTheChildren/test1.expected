7
6
1502
1502
1028
1046
1499
1489
1508
1508
1475
4
2
2
3
2
3
98
194
290
387
